fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/generator/ArbitraryGeneratorContext.java

```diff
@@ -60,6 +60,7 @@ public final class ArbitraryGeneratorContext implements Traceable {
 	private final int generateUniqueMaxTries;
 	private final AtomicReference<CombinableArbitrary<?>> generated =
 		new AtomicReference<>(CombinableArbitrary.NOT_GENERATED);
+	private final boolean enableLoggingFail;
```

Comment:
To prevent the breaking change, could you make an class `ArbitraryGeneratorLoggingContext` that includes `enableLoggingFail`?

Reply:
```java
public final class ArbitraryGeneratorLoggingContext {
    private final boolean enableLoggingFail;
    ...
    public boolean isEnableLoggingFail(){
        return enableLoggingFail;
    }
}
```
```java
public final class ArbitraryGeneratorContext implements Traceable {
    ...
    private final ArbitraryGeneratorLoggingContext loggingContext;
    ...
    public ArbitraryGeneratorLoggingContext getLoggingContext() {
        return this.loggingContext;
    }
...
}
```

```java
public final class BeanArbitraryIntrospector implements ArbitraryIntrospector {
    ...
    @Override
    public ArbitraryIntrospectorResult introspect(ArbitraryGeneratorContext context) {
        ArbitraryGeneratorLoggingContext loggingCotext = context.getLoggingContext();
        ...
        if(loggingContext.isEnableLoggingFail()){
        ...
        }
    }
    ...
}
```

Like this?

Reply:
Yes, indeed